tests/rptest/services/consumer_swarm.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2020 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from dataclasses import dataclass
+from ducktape.tests.test import TestContext
+from ducktape.services.service import Service
+from typing import Optional
+
+import time
+import requests
+
+
+class ConsumerSwarm(Service):
+    EXE = "client-swarm"
+    LOG_PATH = "/opt/remote/var/client-swarm.log"
+
+    logs = {"repeater_log": {"path": LOG_PATH, "collect_default": True}}
+
+    def __init__(self,
+                 context: TestContext,
+                 redpanda,
+                 topic: str,
+                 producers: int,
+                 records_per_producer: int,
+                 log_level="DEBUG",
+                 properties={},
+                 timeout_ms: int = 1000,
+                 compression_type: Optional[str] = None,
+                 compressible_payload: Optional[bool] = None,
+                 min_record_size: Optional[int] = None,
+                 max_record_size: Optional[int] = None,
+                 keys: Optional[int] = None,
+                 unique_topics: Optional[bool] = False,
+                 static_prefix: Optional[bool] = False,
+                 unique_groups=False):
+        super(ConsumerSwarm, self).__init__(context, num_nodes=1)
+        self._redpanda = redpanda
+        self._topic = topic
+        self._producers = producers
+        self._records_per_producer = records_per_producer
+        self._log_level = log_level
+        self._properties = properties
+        self._timeout_ms = timeout_ms
+        self._compression_type = compression_type
+        self._compressible_payload = compressible_payload
+        self._min_record_size = min_record_size
+        self._max_record_size = max_record_size
+        self._keys = keys
+        self._unique_topics = unique_topics
+        self._unique_topics = unique_groups
```

Comment:
this looks bugged

Reply:
Will note and correct in the first PR to follow.